Dart/src/com/jetbrains/lang/dart/sdk/DartConfigurable.java

```diff
@@ -410,6 +418,11 @@ private String getErrorMessage() {
       }
     }
 
+    String sdkVersion = DartSdk.getDartSdk(myProject).getVersion();
```

Comment:
The Dart SDK version should be fetched from the dialog settings higher in the dialog itself so that if a higher version of the SDK is actively selected, users won't see the error message.

Reply:
Done, thanks!